src/sidebar/components/Thread.js

```diff
@@ -73,64 +158,64 @@ function Thread({ thread, threadsService }) {
       ),
     [
       hasAppliedFilter,
+      isReply,
       onToggleReplies,
       thread.annotation,
-      thread.parent,
       thread.replyCount,
       thread.collapsed,
       thread.visible,
     ]
   );
 
   return (
-    <section
-      className={classnames('Thread', {
-        'Thread--reply': thread.depth > 0,
-        'is-collapsed': thread.collapsed,
-      })}
-    >
-      {showThreadToggle && (
-        <div className="Thread__collapse">
-          <div className="Thread__collapse-button-container">
-            <IconButton
-              classes="NonResponsiveIconButton"
-              expanded={!thread.collapsed}
-              icon={toggleIcon}
-              title={toggleTitle}
-              onClick={onToggleReplies}
-              size="medium"
-              variant="light"
-            />
-          </div>
-        </div>
+    <section className="flex" data-testid="thread-container">
+      {isReply && (
+        <ThreadCollapseControl
+          threadIsCollapsed={thread.collapsed}
+          onToggleReplies={onToggleReplies}
+        />
       )}
 
-      <div className="Thread__content">
+      <div
+        className={classnames(
+          // Set a max-width to ensure that annotation content does not exceed
+          // the width of the container
+          'grow max-w-full'
+        )}
+        data-testid="thread-content"
+      >
         {annotationContent}
 
-        {showHiddenToggle && (
-          <>
+        {isHidden && (
+          <div className="space-y-2">
             {!thread.parent && (
-              <ThreadHeader
+              <HiddenThreadCardHeader
                 annotation={thread.annotation}
                 replyCount={thread.replyCount}
                 threadIsCollapsed={thread.collapsed}
               />
             )}
-            <div className="Thread__hidden-toggle-button-container">
+            <div>
               <LabeledButton
                 onClick={() => threadsService.forceVisible(thread)}
               >
                 Show {countHidden(thread)} more in conversation
               </LabeledButton>
             </div>
-          </>
+          </div>
         )}
 
-        {showChildren && (
-          <ul className="Thread__children">
+        {!thread.collapsed && (
+          <ul
+            className={classnames(
+              // Pull this list to the left to bring it closer to the left edge
+              // of the ThreadCard and give more space for nested replies' content
+              '-ml-3'
+            )}
+            data-testid="thread-children"
+          >
             {visibleChildren.map(child => (
-              <li key={child.id}>
+              <li key={child.id} className="mt-2">
```

Comment:
This top margin was previously set on the thread container element, but only for replies. So it seemed more appropriate for each reply-containing `li` to set its top margin instead. This is what defines spacing between the annotations in a thread.